packages/e2e-tests/specs/undo.test.js

```diff
@@ -84,54 +141,121 @@ describe( 'undo', () => {
 		await pressKeyWithModifier( 'primary', 'z' ); // Undo 3rd paragraph text.
 
 		expect( await getEditedPostContent() ).toBe( thirdBlock );
```

Comment:
We should add a check before the first undo here.

Reply:
It should be enough to only test selection after undo right? After typing, we already know for sure where the selection is.

Reply:
>After typing, we already know for sure where the selection is.

It wouldn't hurt to verify that it synced correctly.